hw8/hw8_t02.py

```diff
@@ -0,0 +1,64 @@
+"""
+Task.
+
+Write a wrapper class TableData for database table, that when initialized with database name and
+table acts as collection object (implements Collection protocol).
+Assume all data has unique values in 'name' column.
+So, if presidents = TableData(database_name='example.sqlite', table_name='presidents')
+
+then
+ -  `len(presidents)` will give current amount of rows in presidents table in database
+ -  `presidents['Yeltsin']` should return single data row for president with name Yeltsin
+ -  `'Yeltsin' in presidents` should return if president with same name exists in table
+ -  object implements iteration protocol. i.e. you could use it in for loops::
+       for president in presidents:
+           print(president['name'])
```

Comment:
What's the right way to test this requirement?

Reply:
This is the requirement: 
```
object implements iteration protocol
```

This is the example:
```python
# i.e. you could use it in for loops
for president in presidents:
           print(president['name'])
```